grails-app/controllers/org/pih/warehouse/JsonController.groovy

```diff
@@ -1861,7 +1862,7 @@ class JsonController {
                 }
 
                 response.setHeader("Content-disposition", "attachment; filename=\"Request-Detail-Report.csv\"")
-                render(contentType: "text/csv", text: sw.toString(), encoding: "UTF-8")
+                render(contentType: "text/csv", text: CSVUtils.prependBOMToCSVString(sw.toString()), encoding: "UTF-8")
```

Comment:
make this camelcase for readability
prependBomToCsvString

Reply:
I suggested `prependBOMToCSVString` to match the "convention" we had in the `CSVUtils` (-> `CSV` in the class name and another method), but if you want a camel case, then I'm fine with that.